<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3d_viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="myopengl" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>68</y>
      <width>511</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_value">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>136</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_value">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>166</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_value">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>196</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_add">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>132</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>x++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_sub">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>192</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>z--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_add">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>161</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>y++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_sub">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>161</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>y--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_sub">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>132</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>x--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_add">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>192</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>z++</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zoom_value">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>256</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_sub">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>251</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>%--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_add">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>251</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>%++</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_rot_value">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>316</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_rot_value">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>346</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_rot_value">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>376</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_rot_add">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>311</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>x++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_rot_sub">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>311</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>x--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_rot_add">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>341</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>y++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_rot_sub">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>341</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>y--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_rot_add">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>371</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>z++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_rot_sub">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>371</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>z--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_file_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>56</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_screen">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>586</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Сделать скриншот</string>
    </property>
   </widget>
   <widget class="QComboBox" name="format">
    <property name="geometry">
     <rect>
      <x>805</x>
      <y>585</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.jpeg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.bmp</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="start_gif">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>626</y>
      <width>301</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Записать gif-анимацию преобразования</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_colour">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>416</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать цвет фона</string>
    </property>
   </widget>
   <widget class="QComboBox" name="line_type">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>530</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>сплошная</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>пунктирная</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="line_thickness">
    <property name="geometry">
     <rect>
      <x>683</x>
      <y>506</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="tops_colour">
    <property name="geometry">
     <rect>
      <x>757</x>
      <y>476</y>
      <width>145</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>выбрать цвет</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="tops_size">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>506</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>506</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tops_type">
    <property name="geometry">
     <rect>
      <x>786</x>
      <y>530</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>отсутствуют</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>квадратные</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>круглые</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="filepath_dots">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>941</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите файл.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>106</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Перемещение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>226</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Масштабирование</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>286</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вращение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lines_colour">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>476</y>
      <width>145</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>выбрать цвет</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>603</x>
      <y>456</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Настройка ребер:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>508</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Толщина:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>536</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>456</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Настройка вершин:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>536</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вид:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>590</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>формат:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myopengl</class>
   <extends>QWidget</extends>
   <header>myopengl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
